CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+## Contributing guidelines
+Contributers must sign the CLA. The instructions will be provided to new contributors when their first PR is opened, as a comment.
```

Comment:
Can we say also, "If you are having problems with the CLA, comment `@microsoft-github-policy-service rerun` on your PR to trigger a re-run

Reply:
Otherwise looks great to me